### PR TITLE
Add configuration for audio

### PR DESCRIPTION
**Pull request by nik0**
- Add settings for audio in (mic) configuration
- Add settings for bitrate configuration
- Double click on tabs name in log page will clear the logs

### CODE REVIEW

File: firmware_mod/controlscripts/rtsp-mjpeg [hunk 1]
```diff
@@ -42,7 +42,22 @@ start()
...
+        AUDIOPARAM="-E $AUDIOFORMAT:$AUDIOOUTBR"
+        /system/sdcard/bin/setconf -k h -v "$HWVOLUME" 2>/dev/null
+        /system/sdcard/bin/setconf -k i -v "$SWVOLUME" 2>/dev/null
+        /system/sdcard/bin/setconf -k q -v "$FILTER" 2>/dev/null
+        /system/sdcard/bin/setconf -k l -v "$HIGHPASSFILTER" 2>/dev/null
+    else
+        AUDIOPARAM="-A"
+    fi
+    echo "================== START ===============" >> "$LOGPATH"
+    echo "/system/sdcard/bin/busybox nohup /system/sdcard/bin/v4l2rtspserver-master -fMJPG $RTSPMJPEGOPTS  $AUDIOPARAM" >> "$LOGPATH
```
Comment (Dopeyr) [inline firmware_mod/controlscripts/rtsp-mjpeg]: I think there's a missing end quote here

File: firmware_mod/controlscripts/rtsp-mjpeg [hunk 2]
```diff
@@ -14,7 +14,7 @@ fi
 
 if [ -f /system/sdcard/config/osd.conf ]; then
   . /system/sdcard/config/osd.conf 2>/dev/null
-fi
+fis
```
Comment (Dopeyr) [inline firmware_mod/controlscripts/rtsp-mjpeg]: Think there's a typo here.

Comment (nik0) [reply firmware_mod/controlscripts/rtsp-mjpeg]: yep ...

File: firmware_mod/controlscripts/rtsp-mjpeg (hunk 1 above)
Comment (nik0) [reply firmware_mod/controlscripts/rtsp-mjpeg]: oups ...

**Pull request merged**